doc/README_Generic.md

```diff
@@ -0,0 +1,61 @@
+Generic YaST README File
+========================
+
+YaST Development Documentation
+------------------------------
+
+- [YaST development documentation](http://yast.github.io/documentation.html)
+- [Contribution Guidelines](http://yast.github.io/guidelines.html)
+- [YaST architecture](http://yastgithubio.readthedocs.org/en/latest/architecture)
+- [Development environment](https://en.opensuse.org/openSUSE:YaST:_Preparing_the_Development_Environment)
+
+
+Development Environment
+-----------------------
+Before doing anything useful with the code, you need to setup a development
+environment. Fortunately, this is quite simple, see the [preparing development
+environment](https://en.opensuse.org/openSUSE:YaST:_Preparing_the_Development_Environment)
+documentation.
+
+
+### Extra Development Tools ###
```

Comment:
@ancorgs I have a different POV, shouldn't we merge the Wiki page into our documentation at this README (or other document?) Wiki is not maintained, our docu is.